doc/src/manual/dates.md

```diff
@@ -181,7 +184,7 @@ julia> dt2 = DateTime(2000,2,1)
 2000-02-01T00:00:00
 
 julia> dt - dt2
-381110402000 milliseconds
+381110400000 milliseconds
```

Comment:
checks out - there was 1 in 1998, 1 in 2005, 1 in 2008, and 1 in June 2012, so 2 in that interval